### DISCUSSION TITLE
Displaying Responses in Upper Case using Survey-Text

### DISCUSSION THREAD

**Discussion by awb99cu** [jspsych/jsPsych#1518] [Q&A]
I am conducting a verbal free recall experiment in which participants type their responses after learning a list of words. Currently my retrieval prompt looks like this:

```javascript
           var prompt = "<p>Please type in as many words from the " +
                     "previous list as you can remember.<p>"+
                    "<p>Press 'return' after entering each word.<p>";
                    return [{prompt: prompt, required: true}];
                  }
                };
```

When participants type the words out, I would like their responses to be recorded and displayed onscreen in uppercase lettering (e.g., "window" --> "WINDOW"). Is there a way to do this in JSPsych? Perhaps using javascript's built-in toUpperCase function?

Thank you,
Adam

**Comment by jodeleeuw** [comment]
I'm going to convert this to a discussion.
